python/delta_sharing/reader.py

```diff
@@ -146,11 +150,20 @@ def __to_pandas_kernel(self):
         # Close the file
         json_file.close()
 
+        if (num_files > DeltaSharingReader.FILE_NUMBER_WARNING_THRESH):
+            print("Warning: Queries with a large number of files take a large ammount of time")
```

Comment:
@linzhou-db  is there a better way to display the warning to the user?

Reply:
as discussed offline, we don't need this.